.github/workflows/compatibility.yml

```diff
@@ -4,15 +4,16 @@ on:
   pull_request
 
 env:
-  WC_L2_VERSION: '7.3.0'
-  WP_L2_VERSION: '5.9'
+  WC_MIN_VERSION: '7.3.0'
+  WP_MIN_VERSION: '6.0'
+  PHP_MIN_VERSION: '7.2'
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-  generate-matrix:
+  generate-first-matrix:
```

Comment:
`first` and `second` in this context are not that clear, could we rename them to indicate the purpose instead?

Reply:
Replying to this comment will take me longer than renaming them, but I don't see the point in being more expressive in these names. Each matrix job is immediately followed by the job that needs it, and the two jobs (`woocommerce-compatibility` and  `compatibility-woocommerce-beta`) are doing the same thing. They are 2 separate jobs because we want to allow a part of the testing matrix to fail, if I understood correctly by the comments (by the way, now that I think about it, I'm not sure what is the benefit of splitting the two jobs? The workflow will be shown as a failure if the second "allowed to fail" job actually fails..). I might be missing some history here).

Anyway, will rename them to include the job it will be used in, but wanted to share some thoughts on the topic still

Reply:
I don't have the relevant context either but I just noticed that in the original PR, we had `continue-on-error` set as `true` for the second matrix which was then maybe removed later, [ref](https://github.com/Automattic/woocommerce-payments/pull/1134/files#diff-0ff6bcc45c7c09bfcb5419c83559f190add597ec3b82ad238ddb1d6e43e9517eR82). 

So maybe that was the original thought to allow failures but wasn't considered in some follow up.